### PR TITLE
Newsletters: add 325 (2024-10-18)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/services @bitschmidty 
- [x] Notable merges @Gustavojfe

### CODE REVIEW

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+slug: 2024-10-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+_No significant news this week was found in any of our [sources][optech sources]._
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-10-18-newsletter.md:14]: https://delvingbitcoin.org/t/ln-summit-2024-notes-summary-commentary/1198 just dropped, FYI, if there is time

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Coinbase adds taproot send support:**
+  Coinbase exchange [now supports][coinbase post] user withdrawals (send) to taproot
+  [bech32m][topic bech32] addresses.
+
+- **Dana wallet released:**
+  [Dana wallet][dana wallet github] is a [silent payment][topic silent payments] wallet focused the
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-10-18-newsletter.md:26]:
```suggestion
  [Dana wallet][dana wallet github] is a [silent payment][topic silent payments] wallet focused on the
```

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  donation use case. The developers recommend using [signet][topic signet] and
+  also run a signet [faucet][dana wallet faucet].
+
+- **Bitcoin script editor and visualizer:**
+  The Saving Satoshi [script editor][saving satoshi editor] allows for visually
+  building and executing Bitcoin scripts.
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-10-18-newsletter.md:32]: I am not sure we should include this just yet, I've played with the editor for a few minutes and already found serious bugs in the script interpreter. For example, `OP_0 OP_SHA256` does not hash the empty byte array but the string "0". Additionally, only around half the existing opcodes are implemented, for example most arithmetic and stack opcodes are missing. And there are other issues, for example cleanstack is enforced without explaining why the script "failed".

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  The [DLC][topic dlc]-based platform [announced][dlc markets blog] mainnet
+  availability for its non-custodial trading service.
+
+- **Ashigaru wallet announced:**
+  Ashigaru is a fork of the Samourai Wallet project and the
+  [announcement][ashigaru blog] listed improvements to [batching][scaling
+  payment batching], [RBF][topic rbf] support, and [fee estimation][topic fee estimation].
+
+- **DATUM protocol announced:**
+  The [DATUM mining protocol][datum docs] the allows miners to build candidate blocks as part
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-10-18-newsletter.md:48]:
```suggestion
  The [DATUM mining protocol][datum docs] allows miners to build candidate blocks as part
```

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-beta.5][] is a release candidate (RC) of this library for
+  building wallets and other Bitcoin-enabled applications.  This latest
+  RC "enables RBF by default, updates the bdk_esplora client to retry
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-10-18-newsletter.md:69]:
```suggestion
  RC "enables RBF by default, it also updates the bdk_esplora client to retry
```
Or perhaps:
```suggestion
  RC "enables RBF by default [and] updates the bdk_esplora client to retry
```

File: _posts/en/newsletters/2024-10-18-newsletter.md (hunk 1 above)
Comment (harding) [reply _posts/en/newsletters/2024-10-18-newsletter.md]: Added!

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+    commitment protocol (see next item).
+
+  - **State update protocol:** a proposal was discussed to convert LN's
+    current state update protocol from allowing either side to propose an
+    update at any time to only allowing a one party at a time to propose
+    updates (see Newsletters [#120][news120 simcom] and
+    [#261][news261 simcom]).  Allowing either side to propose updates can
+    result in both side proposing updates simultaneously, which is
+    difficult to reason about and can lead to accidental channel force
+    closures when.  The alternative is for only one party to be in
```
Comment (shesek) [inline _posts/en/newsletters/2024-10-18-newsletter.md:43]: "when" appears out of place

**Comment by Gustavojfe** [comment]
just added merge summaries.

File: _posts/en/newsletters/2024-10-18-newsletter.md (hunk 3 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2024-10-18-newsletter.md:106]: This is good feedback. Will remove for this month.

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+    into the tradeoffs of various possible [PTLC][topic ptlc]
+    implementations was discussed (see [Newsletter #268][news268 ptlc]).
+    A particular focus was the construction of the [signature
+    adaptor][topic adaptor signatures] (e.g. using scripted multisig
+    versus scriptless [MuSig2][topic musig]) and its effect on the
+    commitment protocol (see next item).
+
+  - **State update protocol:** a proposal was discussed to convert LN's
+    current state update protocol from allowing either side to propose an
+    update at any time to only allowing a one party at a time to propose
```
Comment (jirijakes) [inline _posts/en/newsletters/2024-10-18-newsletter.md:38]:
```suggestion
    update at any time to only allowing one party at a time to propose
```

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+[pod323 eclair]: /en/podcast/2024/10/08/#eclair-2848-transcript
+[osuntokun summary]: https://delvingbitcoin.org/t/ln-summit-2024-notes-summary-commentary/1198
+[osuntokun notes]: https://docs.google.com/document/d/1erQfnZjjfRBSSwo_QWiKiCZP5UQ-MR53ZWs4zIAVcqs/edit?tab=t.0#heading=h.chk08ds793ll
+[news268 ptlc]: /en/newsletters/2023/09/13/#ln-messaging-changes-for-ptlcs
+[news120 simcom]: /en/newsletters/2020/10/21/#simplified-htlc-negotiation
+[news261 simcom]: /en/newsletters/2023/07/26/#simplified-commitments
+[zmnscpxj superscalar]: https://delvingbitcoin.org/t/superscalar-laddered-timeout-tree-structured-decker-wattenhofer-factories/1143
+[news309 feasible]: /en/newsletters/2024/06/28/#estimating-the-likelihood-that-an-ln-payment-is-feasible
+[news310 mining]:/en/newsletters/2024/07/05/#bitcoin-core-30200
+[news323 fees]: /en/newsletters/2024/10/04/#eclair-2861
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-10-18-newsletter.md:228]: these two 323 references are reversed

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+    adaptor][topic adaptor signatures] (e.g. using scripted multisig
+    versus scriptless [MuSig2][topic musig]) and its effect on the
+    commitment protocol (see next item).
+
+  - **State update protocol:** a proposal was discussed to convert LN's
+    current state update protocol from allowing either side to propose an
+    update at any time to only allowing a one party at a time to propose
+    updates (see Newsletters [#120][news120 simcom] and
+    [#261][news261 simcom]).  Allowing either side to propose updates can
+    result in both side proposing updates simultaneously, which is
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-10-18-newsletter.md:41]:
```suggestion
    result in both sides proposing updates simultaneously, which is
```

File: _posts/en/newsletters/2024-10-18-newsletter.md (hunk 6 above)
Comment (LarryRuane) [reply _posts/en/newsletters/2024-10-18-newsletter.md:43]: I think it should just be removed.

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+  channel quiescence (see Newsletter [#309][news309 quiescence]), to conform to
+  the latest splicing protocol as proposed in [BOLTs #1160][], which requires
+  nodes to use the quiescence protocol during splicing. Previously, splicing was
+  allowed under a less formal mechanism, where splice messages were permitted if
+  the commitments were already quiescent, acting as a "poor man's" version of
+  channel quiescence.
+
+- [LDK #3235][] adds a `last_local_balance_msats` field to the
+  `ChannelForceClosed` event, which gives the local balance of a node in
+  millisatoshis (msats) just before the channel was force closed, allowing users
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-10-18-newsletter.md:181]:
```suggestion
  millisatoshis (msats) just before the channel was force-closed, allowing users
```

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+  to know how many msats they lost due to rounding.
+
+- [LND #8183][] adds the optional `CloseTxInputs` field to the
+  `chanbackup.Single` structure in the [static channel backup][topic static
+  channel backups] (SCB) file, to store the inputs required to generate force
+  close transactions. This allows users to manually retrieve funds when a peer
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-10-18-newsletter.md:187]:
```suggestion
  channel backups] (SCB) file, to store the inputs required to generate force-close
  transactions. This allows users to manually retrieve funds when a peer
```

File: _posts/en/newsletters/2024-10-18-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+  is offline using the `chantools scbforceclose` command as a last resort
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-10-18-newsletter.md:188]:
```suggestion
  is offline using the `chantools scbforceclose` command as a last-resort
```

**Comment by harding** [comment]
Made edits for all feedback, thanks @bitschmidty @vostrnad @shesek @jirijakes @LarryRuane !  Added lede, checked for updated releases/RCs (none found), added topic entries, and reviewed excellent sections from @bitschmidty and @Gustavojfe.  Thanks everyone!

**Review by bitschmidty — COMMENTED** [review]
ACK lede, topics. Squashed. Thank you all! 🚀

**Pull request merged**